selfdrive/car/gm/interface.py

```diff
@@ -156,6 +159,8 @@ def update(self, c, can_strings):
       events.add(EventName.resumeRequired)
     if self.CS.pcm_acc_status == AccState.FAULTED:
       events.add(EventName.controlsFailed)
+    if ret.vEgo < self.CP.minSteerSpeed:
+      events.add(car.CarEvent.EventName.belowSteerSpeed)
```

Comment:
Is this necessary? Due to:
```
    if ret.vEgo < self.CP.minEnableSpeed:
      events.add(EventName.belowEngageSpeed)
```
We won't be able to engage below `minEnableSpeed` anyway (which is currently set to 18mph for Volt).

Change `minEnableSpeed` to 7mph instead? (This is true for Volt, but I'm sure about other GM vehicles).
FWIW, 2018's can engage at 0mph, but 2017s and earlier cannot.

Reply:
This covers the case where you're already engaged and you go below the minSteerSpeed.

Reply:
Ok, gotcha. I think a warning when < 7mph is fair (although arguably distracting in stop-n-go traffic).